recipes/quast/meta.yaml

```diff
@@ -49,11 +49,13 @@ requirements:
     - sambamba
 
 test:
+  source_files:
```

Comment:
Is test_data really needed?

Reply:
It is needed for running quast --test and metaquast--test executed below. However, it is not needed for installation itself, so it is not installed by default. So, here I ask conda to copy the data to temporary dir before testing (if I understand the syntax correctly).